src/Text/Layout/Table/Style.hs

```diff
@@ -5,123 +5,195 @@ module Text.Layout.Table.Style where
-                , headerTopH   :: Char
-                , headerV      :: Char
-                , groupV       :: Char
-                , groupSepH    :: Char
-                , groupSepC    :: Char
-                , groupSepLC   :: Char
-                , groupSepRC   :: Char
-                , groupTopC    :: Char
-                , groupTopL    :: Char
-                , groupTopR    :: Char
-                , groupTopH    :: Char
-                , groupBottomC :: Char
-                , groupBottomL :: Char
-                , groupBottomR :: Char
-                , groupBottomH :: Char
+                { headerSepH   :: String
+                , headerSepLC  :: String
+                , headerSepRC  :: String
+                , headerSepC   :: String
+                , headerTopL   :: String
+                , headerTopR   :: String
+                , headerTopC   :: String
+                , headerTopH   :: String
+                , headerL      :: String
+                , headerR      :: String
+                , headerC      :: String
+                , groupL       :: String
+                , groupR       :: String
+                , groupC       :: String
+                , groupSepH    :: String
+                , groupSepC    :: String
+                , groupSepLC   :: String
+                , groupSepRC   :: String
+                , groupTopC    :: String
+                , groupTopL    :: String
+                , groupTopR    :: String
+                , groupTopH    :: String
+                , groupBottomC :: String
+                , groupBottomL :: String
+                , groupBottomR :: String
+                , groupBottomH :: String
                 }
 
+-- | Remove the top, bottom, left, and right borders from a 'TableStyle'.
+withoutBorders :: TableStyle -> TableStyle
+withoutBorders = withoutTopBorder . withoutBottomBorder . withoutLeftBorder . withoutRightBorder
+
+-- | Remove the top border from a 'TableStyle'.
+withoutTopBorder :: TableStyle -> TableStyle
+withoutTopBorder ts = ts { headerTopL = "", headerTopR = "", headerTopC = "", headerTopH = ""
+                         , groupTopL = "", groupTopR = "", groupTopC = "", groupTopH = ""}
+
+-- | Remove the bottom border from a 'TableStyle'.
+withoutBottomBorder :: TableStyle -> TableStyle
+withoutBottomBorder ts = ts { groupBottomC = "", groupBottomL = "", groupBottomR = "", groupBottomH = ""}
+
+-- | Remove the left border from a 'TableStyle'.
+withoutLeftBorder :: TableStyle -> TableStyle
+withoutLeftBorder ts = ts { headerTopL = "", headerSepLC = "", headerL = ""
+                          , groupL = "", groupSepLC = "", groupTopL = "", groupBottomL = ""}
+
+-- | Remove the right border from a 'TableStyle'.
+withoutRightBorder :: TableStyle -> TableStyle
+withoutRightBorder ts = ts { headerTopR = "", headerSepRC = "", headerR = ""
+                           , groupR = "", groupSepRC = "", groupTopR = "", groupBottomR = ""}
```

Comment:
I am a bit allergic to a lack of spaces, but nothing personal.  I must beg you to insert a space before `}` or preferably add a newline and align with `,` and `{`.